frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -47,7 +47,7 @@ export const EmbedHomepage = () => {
     getPlan(getSetting(state, "token-features")),
   );
 
-  const defaultTab = useMemo(() => {
+  const initialTab = useMemo(() => {
```

Comment:
I renamed this to be consistent with the analytics, where the name "default" made less sense